src/cpp/fastdds/xtypes/type_representation/TypeObjectUtils.cpp

```diff
@@ -2094,6 +2071,7 @@ EquivalenceKind TypeObjectUtils::get_map_component_equiv_kind_for_consistency(
     {
         if (is_key)
         {
+            // indirecr hash for key is not allowed
```

Comment:
Typo: indirect

Reply:
fixed in [9449b0e](https://github.com/eProsima/Fast-DDS/pull/4496/commits/9449b0eee57a38c4f9b38a51c31f8bf24d81935f)